mmv1/products/securitycenter/NotificationConfig.yaml

```diff
@@ -94,6 +96,9 @@ properties:
       - !ruby/object:Api::Type::String
         name: filter
         required: true
+        send_empty_value: true
+        validation: !ruby/object:Provider::Terraform::Validation
+          function: 'validation.StringLenBetween(0,1024)'
```

Comment:
```suggestion
```

Adding this new validation can theoretically break users -- we should let it be handled by the API

Reply:
Addressed